FBSnapshotTestCase/FBSnapshotTestCase.modulemap

```diff
@@ -0,0 +1,6 @@
+framework module FBSnapshotTestCase {
+    umbrella header "FBSnapshotTestCase.h"
+
+    export *
+    module * { export * }
```

Comment:
@pietbrauer: You'd only need a modulemap, if you want to access from Swift a specific part of the Objective-C interface, which shouldn't be part of the regular 'public' API. All involved Objective-C headers must be public though, but you can avoid at least that they are included by default.